scripts/utils/train_helper.py

```diff
@@ -200,7 +203,8 @@ def get_fcp_trained_w_selfplay_types(args,
                                                         train_types=fcp_w_sp_train_types,
                                                         eval_types=fcp_w_sp_eval_types['generate'],
                                                         )
-    seed, h_dim = 2602, 256
+    seed = args.trainhelper_seed
+    h_dim = args.trainhelper_h_dim
```

Comment:
I think all these seed,h_dim variables should be deleted and moved to `set_input` function in `train_agents.py` (I also mentioned it in my comment on arguments.py)